runtime/src/lib.rs

```diff
@@ -209,7 +209,7 @@ pub const MAXIMUM_BLOCK_WEIGHT: Weight = Weight::from_parts(
 
 /// Maximum number of blocks simultaneously accepted by the Runtime, not yet included into the
 /// relay chain.
-const UNINCLUDED_SEGMENT_CAPACITY: u32 = 3;
```

Comment:
Just out of curiosity, what was the reason behind changing this?

Reply:
As of right now, I put the velocity to 1 to avoid producing more than a single block per slot since in Moonbeam they saw a lot of reorgs when increasing this. A capacity of 2-3 is recommended and either way the real benefits come from the relay chain slot duration set to 6 seconds. Once we are ready, we can tackle this and optimize these parameters so we can produce multiple blocks per slot and increase capacity.

```rust
/// Maximum number of blocks simultaneously accepted by the Runtime, not yet included into the
/// relay chain.
const UNINCLUDED_SEGMENT_CAPACITY: u32 = 2;
/// How many parachain blocks are processed by the relay chain per parent. Limits the number of
/// blocks authored per slot.
const BLOCK_PROCESSING_VELOCITY: u32 = 1;
/// Relay chain slot duration, in milliseconds.
const RELAY_CHAIN_SLOT_DURATION_MILLIS: u32 = 6000;
```